packages/integration-tests/api/api.js

```diff
@@ -394,6 +394,121 @@ async function addProviderApi({ prefix, provider }) {
   });
 }
 
+/**
+ * Fetch a list of providers from the Cumulus API
+ *
+ * @param {Object} params - params
+ * @param {string} params.prefix - the prefix configured for the stack
+ * @returns {Promise<Object>} - the list of providers fetched by the API
+ */
+async function getProviders({ prefix }) {
+  const payload = await callCumulusApi({
+    prefix: prefix,
+    functionName: 'ApiProvidersDefault',
+    payload: {
+      httpMethod: 'GET',
+      resource: '/providers',
+      path: '/providers'
+    }
+  });
+
+  return JSON.parse(payload.body);
+}
+
+/**
+ * Fetch a provider from the Cumulus API
+ *
+ * @param {Object} params - params
+ * @param {string} params.prefix - the prefix configured for the stack
+ * @param {string} params.providerId - the ID of the provider to fetch
+ * @returns {Promise<Object>} - the provider fetched by the API
+ */
+async function retrieveProvider({ prefix, providerId }) {
```

Comment:
`getProvider` would be a slightly more intuitive name, since this is using the "GET" method.

Reply:
I can change the names but I was trying to avoid getProvider and getProviders being so close